.scripts/mark_file/main.py

```diff
@@ -0,0 +1,111 @@
+import polib
+import os
+import datetime
+import requests
+
+
+def entry_check(pofile):
+    lines_tranlated = len(pofile.translated_entries())
+    lines_untranlated = len(pofile.untranslated_entries())
+
+    if lines_tranlated == 0:
+        result = "❌"
+    elif lines_untranlated == 0:
+        result = "✅"
+    else:
+        lines_all = lines_tranlated + lines_untranlated
+        progress = lines_tranlated / lines_all
+        progress = round(progress*100, 2)
+        result = f"Ongoing, {str(progress)} %"
+
+    return result
+
+
+def get_github_issue():
+    NUMBER_OF_ISSUES = 100
+
+    url = f"https://api.github.com/repos/python/python-docs-zh-tw/issues?per_page={NUMBER_OF_ISSUES}"
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28"
+    }
+    r = requests.get(url=url, headers=headers)
+    result = r.json()
+
+    result_list = []
+    for issue in result:
+        title = issue['title'].split(" ")
+
+        if len(title) < 2:
+            continue
+        if title[0] != "翻譯" and title[0].lower() != "translate":
+            continue
+        if issue["assignee"] is None:
+            continue
+
+        filename = title[1]
+        if filename[0] == "`":
+            filename = filename[1:]
+        if filename[-1] == "`":
+            filename = filename[:-1]
+
+        filename = filename.split("/")
+        if len(filename) < 2:
+            continue
+        if filename[1][-3:] != ".po":
+            filename[1] += ".po"
+
+        result_list.append([filename, issue["assignee"]["login"]])
+
+    return result_list
+
+
+def format_line_file(filename, result):
+    tmp = f"  - {filename}"
+    tmp = f"{tmp}{'-' * (40-len(tmp))}{result}\r"
+    return tmp
```

Comment:
- use buildin func instead
- why not using `\r\n`?


```suggestion
    return f"  - {filename.ljust(37, '-')}{result}\r\n"
```

Reply:
I already forget what I done then, I'll fix here up lol